packages/hash/frontend/src/pages/shared/auth-layout.tsx

```diff
@@ -56,3 +57,28 @@ export const AuthLayout: VoidFunctionComponent<AuthLayoutProps> = ({
     </div>
   );
 };
+
+export type AuthModalLayoutProps = {
+  onClose?: () => void;
+  show: boolean;
+  loading?: boolean;
+  children: ReactNode;
+};
+
+export const AuthModalLayout: VFC<AuthModalLayoutProps> = ({
```

Comment:
We can agree to export only one component and its props from any tsx file. The component name should match the file name. The file can have as many helper components as necessary, but the must remain private (not exported at all).

Happy for us to change this later if you want.

Reply:
I agree. Addressed in [8c19e64](https://github.com/hashintel/hash/pull/468/commits/8c19e64f44905c778cdeca75b7c8d919b5e779b3)